akasha-vladan-main/css/style.scss

```diff
@@ -0,0 +1,76 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;700&display=swap');
+
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    font-family: 'Open Sans', sans-serif;
+
+}
+
+.title-wrap {
+    background-color: black;
+    padding-top: 15px;
+    padding-bottom: 15px;
+}
+
+
+#title{
+    color: white;
+    text-align: center;
+}
```

Comment:
please follow the CSS best practices by adding a space between "title" and "{"  
Kindly try to follow the CSS best practices your new boss will highly thank you for it.